src/runner/provider/qti.js

```diff
@@ -80,6 +81,11 @@ var qtiItemRuntimeProvider = {
             try {
                 //render item html
                 elt.innerHTML = this._item.render({});
+
+                // apply RTL layout according to item language
+                const $item = $(elt).find('.qti-item');
+                const itemLang = $item.attr('lang');
+                $item.attr('dir', locale.getLanguageDirection(itemLang));
```

Comment:
**thought:** I wonder if the language direction should not also be taken from the item attribute, as specified by the QTI standard